src/java.desktop/share/classes/javax/swing/JFileChooser.java

```diff
@@ -1412,7 +1412,8 @@ public boolean isMultiSelectionEnabled() {
 
     /**
      * Returns true if hidden files are not shown in the file chooser;
-     * otherwise, returns false.
+     * otherwise, returns false. The default value of this property is derived
+     * from native Operating System, unless set explicitly by the programmer.
```

Comment:
Is this true on all platforms ? The bug report seems concerned only about windows.
Should the wording be
"The default value of this property may be derived from the underlying operating system"

ie "is" -> "may be" (and some other slight wording change).

If we REALLY will always get it from the O/S, including on Linux, macOS and whatever else .. then it is OK otherwise the new wording would be better.

"unless set explicitly by the programmer."

hmm, well if you set it, it isn't the default, so there's no need to say that, is there ?

Reply:
> If we REALLY will always get it from the O/S, including on Linux, macOS and whatever else .. then it is OK otherwise the new wording would be better.

Haven't checked in Linux or macOS...... Yeah, I think no need to explicitly mention about setting it by programmer...... Sure will update the PR based on your feedback @prrace .......